docs/developers/README.md

```diff
@@ -280,3 +280,127 @@ status:
 ```
 
 Instead of hardcoding the path to the result file, the user can also use a variable. So `/tekton/results/current-date-unix-timestamp` can be replaced with: `$(results.current-date-unix-timestamp.path)`. This is more flexible if the path to result files ever changes.
+
+## How task results can be used in pipeline's tasks
+
+Now that we have tasks that can return a result, the user can refer to a task result in a pipeline by using the syntax
+`$(tasks.<task name>.results.<result name)`. This will substitute the task result at the location of the variable.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
```

Comment:
I suggest copying this into an example in `examples/pipelineruns` and linking there from here.  Maybe include just the most relevant parts as snippets to illustrate your text description (maybe just the "params" and "results" portions?)  This full pipeline is quite long I think.

Another nice benefit will be that the example file will work as an e2e test as well.

Reply:
done.